provision/setup-automatic-updates

```diff
@@ -5,7 +5,7 @@ set -e
 apt-get update
 apt-get install -y unattended-upgrades curl
 
-tee /etc/apt/apt.conf.d/20auto-upgrades <<EOF > /dev/null
+cat <<EOF > /etc/apt/apt.conf.d/20auto-upgrades
```

Comment:
This file just swaps `tee` for the more usual `cat`. This was my fault initially in not giving clear instructions